docs/sources/clients/fluentd/_index.md

```diff
@@ -180,7 +180,7 @@ This plugin automatically adds a `fluentd_thread` label with the name of the buf
 ### url
 
 The url of the Loki server to send logs to.  When sending data the publish path (`../api/loki/v1/push`) will automatically be appended.
```

Comment:
Please also update this prose, even though your PR did not originally touch this prose.